packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -338,7 +338,7 @@ module.exports = {
       },
       {
         selector: 'ForOfStatement',
-        message: 'iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.',
+        message: 'iterators/generators may require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, imperative loops should be avoided in favor of array iterations.',
```

Comment:
what loops are non-imperative?

Reply:
Array.map, Array.reduce, Array.filter etc are non-imperative approaches

Reply:
Those aren't loops, those are iterations.

Reply:
Yes

Reply:
how about this:
```suggestion
        message: 'iterators/generators require regenerator-runtime in older browsers/engines, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.',
```